rogue-thi-app/components/RoomMap.jsx

```diff
@@ -26,7 +26,7 @@ const SPECIAL_ROOMS = {
 const SEARCHED_PROPERTIES = [
   'Gebaeude',
   'Raum',
-  'Funktion'
+  'Funktion_de'
```

Comment:
Is there a reason why we don't offer support for english language in the search? 
The geojson supports `Funktion_en`